services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/OrchestratorPlugin.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.osc.orchestrator;
+
+import org.eclipse.osc.services.ocl.loader.Ocl;
+
+/**
+ * This interface describes orchestrator plugin in charge of interacting with backend fundamental APIs.
+ */
+public interface OrchestratorPlugin {
+
+    /**
+     * Register a managed service using the provided OCL descriptor.
+     *
+     * @param ocl the OCL model describing the managed service.
+     * @return the service ID on the CSP infrastructure.
+     */
+    String registerManagedService(Ocl ocl);
+
+    /**
+     * Start (exposing the managed service to the users) the managed service.
+     *
+     * @param sid the service ID to start.
+     */
+    void startManagedService(String sid);
```

Comment:
would it be possible to start/stop the managed service? seems there's some security issues to allow us to login to users guest os to do such operations.

Reply:
ignore this, if it means just exposing/hiding to users from osc layer.

Reply:
Yes, correct. As said in previous comment, OSC is not visible for end users. The OSC runtime (and so orchestrator) is on CSP infra. It's actually the CSP "admin" service that interact with OSC orchestrator.